### PR TITLE
filogic: add support for ASUS RT-AX59U

### PR DESCRIPTION
**Pull request by liushiyou006**
Hardware
------------
SOC:   MediaTek MT7986
RAM:   512MB DDR4
FLASH: 128MB SPI-NAND (Winbond W25N01GV)
WIFI:  Mediatek MT7986 DBDC 802.11ax 2.4/5 GHz
ETH:   MediaTek MT7531 Switch
UART:  3V3 115200 8N1 (Pinout silkscreened / Do not ocnnect VCC)

Installation
-----------------------------------------------------------
 Install openwrt image:
1. Download the OpenWrt initramfs image. 
    Copy the image to a TFTP server reachable at 192.168.1.70/24. Rename the image to rtax59u.bin.

2. Connect the PC with TFTP server to the RT-AX59U. 
   Set a static ip on the ethernet interface of your PC. 
    (ip address: 192.168.1.70, subnet mask:255.255.255.0) 
   Conect to the serial console,interrupt the autoboot process by pressing '4' when prompted.

3. Download & Boot the OpenWrt initramfs image as below:

   $: setenv ipaddr 192.168.1.1 
   $: setenv serverip 192.168.1.70 
   $: tftpboot 0x46000000 rtax59u.bin 
   $: bootm 0x46000000

4. Wait for OpenWrt to boot. Transfer the sysupgrade image to the device using scp and install using sysupgrade.

   $:  sysupgrade -n <path-to-sysupgrade.bin>
-------------------------------------------------------------------------------------
 Revert to stock firmware:
1: Download the rt-ax59u firmware from ASUS official website. 
   Save the firmware to tftp server directory and rename  to RT-AX59U.trx

2: Connect the PC with TFTP server to the RT-AX59U.
   Set a static ip on the ethernet interface of your PC.
     (ip address: 192.168.1.70, subnet mask:255.255.255.0)

3: Conect to the serial console,  power on again,  interrupt the autoboot process by pressing '4' when prompted.
    Then enter command:
    $: ubi remove linux
    $: ubi remove jffs2
    $: ubi remove rootfs
    $: ubi remove rootfs_data
    $: ubi create linux 0x45fe000
    $: reset
    then the dut will reboot,interrupt the autoboot process by pressing '2:Load System code then write to Flash via TFTP.' when prompted.      
             when see : Warning!! Erase Linux in Flash then burn new one. Are you sure?(Y/N)
             $: enter y
    you will see the follow, type enter directly:
        Input device IP (192.168.1.1) ==:
        Input server IP (192.168.1.70) ==:
        Input Linux Kernel filename (RT-AX59U.trx) ==:

4: wait for the device run up

### CODE REVIEW

**Comment by Borromini** [comment]
Filogic stuff that's available in Europe - cool!

> you will see the follow, type enter directly:

I would rephrase this as '_hit_ Enter' to avoid confusion. People might actually start _typing_ 'enter'.

**Comment by liushiyou006** [comment]
Dear  Borromini:

As you said in the previous letter "I would rephrase this as '*hit* Enter'
to avoid confusion. People might actually start *typing* 'enter'."
So about commit message doesn't need me to adjust, right?

However, I see the following problem from github：
Do I need to make adjustments about the following errors and how?
[image: 1.jpg]
[image: 2.jpg]
[image: 3.jpg]

Thank you very much.


Borromini ***@***.***> 于2023年6月10日周六 01:30写道：

**Comment by liushiyou006** [comment]
Hello:
How to fix the following issue:

Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: shiyouliu <liushiyou006@gmail.com>')

**Comment by Borromini** [comment]
@liushiyou006 In git, you can do `git commit --amend -s` to add your signoff. You can use --amend to rephrase the commit message as well.

Why are you closing this PR?

**Comment by liushiyou006** [comment]
Dear Borromini:

Sorry, I didn't know if this command : git commit --amend -s worked at
first.
I have created a new PR with signoff.
Thank you.

Borromini ***@***.***> 于2023年6月13日周二 13:25写道：
